backend/config/exaquery_dashboarding_tools.yaml

```diff
@@ -1,10 +1,10 @@
-name: exaquery_metabase
+name: exaquery_dashboarding_tools
```

Comment:
Shouldn't this be a a separate YAML file for Looker and keep this for metabase?

Reply:
It does not make sense, cause the whole code is going to be 90% the same... the only arguable thing here would be that we look into `.bi` schema and we could have it in the default schema so we are making it more generic

Reply:
Also as far as I know, the current solution does not read from 2 different paths and merge the results ( I might be wrong tho' @sztanko ?)